src/logic/and.js

```diff
@@ -1,16 +1,13 @@
-/** @license ISC License (c) copyright 2017 original and current authors */
+/** @license ISC License (c) copyright 2018 original and current authors */
```

Comment:
This was added in 2017, should keep the original year

Reply:
This is reverted now